### ISSUE TITLE
uniformiv covers only vector of size <= 4

### ISSUE THREAD

**Issue by niwtr** [3b/cl-opengl#104]
https://github.com/3b/cl-opengl/blob/e2d83e0977b7e7ac3f3d348d8ccc7ccd04e74d59/gl/opengl.lisp#L714

In gl:uniformiv, the function only covers vectors of size 4 or lesser. For vectors of more elements, the call is **silently** ignored. This should be confusing sometimes :)

**Comment by 3b** [comment]
yeah, probably would be reasonable to use `ecase` instead for those functions dispatching on `length` like that.
